docs/running-on-kubernetes.md

```diff
@@ -0,0 +1,512 @@
+---
+layout: global
+title: Running Spark on Kubernetes
+---
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+Spark can run on clusters managed by [Kubernetes](https://kubernetes.io). This feature is currently experimental.
+
+# Prerequisites
+
+* A spark distribution with Kubernetes.
+* A running Kubernetes cluster with access configured to it using
+[kubectl](https://kubernetes.io/docs/user-guide/prereqs/).  If you do not already have a working Kubernetes cluster,
+you may setup a test cluster on your local machine using
+[minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
+  * We recommend that minikube be updated to the most recent version (0.19.0 at the time of this documentation), as some
```

Comment:
Minikube version mentioned here needs to be updated.

Reply:
@ifilonenko,  I think you did some recent testing with minikube. which version should we recommend for folks running spark/k8s?